packages/x-charts/src/SparkLineChart/SparkLineChart.tsx

```diff
@@ -199,14 +199,7 @@ const SparkLineChart = React.forwardRef(function SparkLineChart(
         axisHighlight?.y === 'none'
       }
     >
-      {plotType === 'bar' && (
-        <BarPlot
-          skipAnimation
-          slots={slots}
-          slotProps={slotProps}
-          sx={{ shapeRendering: 'auto' }}
-        />
-      )}
+      {plotType === 'bar' && <BarPlot skipAnimation slots={slots} slotProps={slotProps} />}
```

Comment:
Why was this changed? 🤔

Reply:
Because of this. 🙈 
![Screenshot 2024-12-20 at 12 17 16](https://github.com/user-attachments/assets/819e792e-2387-479c-8717-f088cb4066b0)